checker/src/org/checkerframework/checker/index/upperbound/UpperBoundAnnotatedTypeFactory.java

```diff
@@ -685,9 +722,7 @@ private void addAnnotationForLiteralPlus(
                 return;
             }
             if (val < -1) {
-                if (nonLiteralType.hasAnnotation(LTLengthOf.class)
-                        || nonLiteralType.hasAnnotation(LTOMLengthOf.class)
-                        || nonLiteralType.hasAnnotation(LTEqLengthOf.class)) {
+                if (isLTL(nonLiteralType) || nonLiteralType.hasAnnotation(LTEqLengthOf.class)) {
```

Comment:
I found the use of isLTL helpful above (thanks!), but here I don't think it buys as much, and is surprising because LTEqLength looks like something isLTL would handle.  Another way to think of this is:  the previous code could be read on its own.  This code requires the reader to look elsewhere in order to understand it, and at the savings of just one clause (from 3 to 2 clauses in the or statement).